backend/src/BuiltinCli/Libs/File.fs

```diff
@@ -90,20 +90,20 @@ let fns : List<BuiltInFn> =
       deprecated = NotDeprecated }
 
 
-    { name = fn "append" 0
+    { name = fn "appendText" 0
```

Comment:
Would be good to start establishing some conventions around this sort of thing (string vs bytes, naming, etc)